### PR TITLE
Add run_tmux_cmd() to execute some shell-command in tmux

### PR DESCRIPTION
**Pull request by Yingshun**
Some cases need to run command line in tmux, so add a method.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
**Test result:**
` (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_network.ovs_br.without_postcopy: PASS (316.58 s)`

File: virttest/utils_tmux.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+def run_tmux_cmd(cmd, session=None):
+    """
+    Execute shell-commands in tmux on localhost or remote
+
+    :param cmd: The command line to run
+    :param session: The session object to the host
+    """
+    if not utils_package.package_install("tmux", session):
+        logging.error("Failed to install required package - tmux!")
```
Comment (chunfuwen) [inline virttest/utils_tmux.py:19]: If tmux is necessary package , better raise exception directly here.

Comment (Yingshun) [reply virttest/utils_tmux.py:19]: Actually, I didn't not raise an exception here on purpose, the output of tmux command line will show the details.

Comment (chunfuwen) [reply virttest/utils_tmux.py:19]: a little confused here, if tmux is not installed successfully and program doesn't terminate here, that means utils_misc.cmd_status_output will be executed without tmux command existence

Comment (Yingshun) [reply virttest/utils_tmux.py:21]: Yes, I'd like to check the result of utils_misc.cmd_status_output's return.

File: virttest/utils_tmux.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Execute shell-commands in tmux on localhost or remote
+
+    :param cmd: The command line to run
+    :param session: The session object to the host
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: command status and output
+    """
+    if not utils_package.package_install("tmux", session):
+        logging.error("Failed to install required package - tmux!")
+    tmux_cmd = 'tmux -c "{}"'.format(cmd)
```
Comment (dzhengfy) [inline virttest/utils_tmux.py:22]: The target is to add 'tmux -c' before a command. So I do not think we need to add a new function. What we need to do is only to combine 'tmux -c' with your target command together.

Comment (dzhengfy) [reply virttest/utils_tmux.py:22]: Generally, I indent to close this PR.

Comment (Yingshun) [reply virttest/utils_tmux.py:22]: Ok, since it's a simple function, I will do it in my tp cases.

**Pull request closed without merging**